tidb-cloud/tidb-cloud-changefeed-states.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Changefeed States
+summary: Learn the meaning of each changefeed state in TiDB Cloud.
+---
+
+# Changefeed States
+
+A [changefeed](/tidb-cloud/changefeed-overview.md) is a replication task in TiCDC, which replicates the data change logs of specified tables in a TiDB cluster to the designated downstream. You can run and manage multiple changefeeds in a TiCDC cluster.
+
+The state of a replication task represents the running status of the replication task. During the running of TiCDC, replication tasks might fail with errors, be manually paused, resumed, or reach the specified `TargetTs`. These behaviors can lead to the change of the replication task state. This document describes the states of TiCDC replication tasks and the transfer relationships between states.
+
+The states in the preceding state transfer diagram are described as follows:
```

Comment:
There is no diagram.Only status description.

Reply:
fixed